modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -73,13 +73,20 @@ class Message extends Component<Renderer.Message> {
       'store'
     ])
 
+    const childIsSessionReset = wrapped.type === 'session_reset'
+    const child = wrapped && (
+      <Message {...sanitizedProps} keyboard={Keyboard} noBubble={!childIsSessionReset} payload={wrapped} />
+    )
+    if (childIsSessionReset) {
+      return child
+    }
```

Comment:
This particular logic breaks the possible cascading of wrappers. I see two more natural solutions:
1. In MessageGroup, prevent wrapping if type = session_reset, or
2. In Message, in the render_session_reset, force a div with the bpw-session_reset class